packages/hashi/test/kolibri.spec.js

```diff
@@ -0,0 +1,98 @@
+// import flatMap from 'lodash/flatMap';
+import Mediator from '../src/mediator';
+import Kolibri from '../src/kolibri';
+
+describe('the kolibri hashi shim', () => {
+  let kolibri;
+  let mediator;
+
+  beforeEach(() => {
+    mediator = new Mediator(window);
+    kolibri = new Kolibri(mediator);
+    kolibri.__setShimInterface();
+  });
+  describe('__setShimInterface method', () => {
+    it('should set kolibri shim property', () => {
+      expect(kolibri.shim).not.toBeUndefined();
+    });
+  });
+  describe('iframeInitialize method', () => {
+    it('should set a "kolibri" property on object', () => {
+      const obj = {};
+      kolibri.iframeInitialize(obj);
+      expect(obj.kolibri).toEqual(kolibri.shim);
+    });
+  });
+
+  describe('getContentById method', () => {
+    let response, client;
+    beforeEach(function() {
+      response = { data: [{ testing: 'testing' }] };
+      client = jest.fn().mockResolvedValue(response);
+      kolibri.shim.client = client;
+    });
+    it('should be called once', () => {
+      kolibri.shim.getContentById().then(() => {
+        expect(client).toHaveBeenCalledTimes(1);
+      });
+    });
+    it('should return a promise that resolves to a ContentNode object', () => {
+      expect(kolibri.shim.getContentById()).toBeInstanceOf(Promise);
+      kolibri.shim.getContentById().then(() => {
+        expect(client).toBeInstanceOf(Object);
```

Comment:
We're not asserting anything about the returned data here, only about `client`.

Should return an argument from the function you're chaining off then, and make an assertion that it's the data from the response.